packages/jest-config/package.json

```
@@ -9,6 +9,9 @@
   "license": "MIT",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "peerDependencies": {
+    "ts-node": "^9.0.0"
```

I think we'll want to make this optional via [`peerDependencyMeta`](https://github.com/yarnpkg/rfcs/blob/master/accepted/0000-optional-peer-dependencies.md) so that folks can bring their own `ts-node` if they actually want to use a `ts` config.

---

```suggestion
    "ts-node": ">= 9.0.0"
```

This might be a better constraint as it'll allow for majors higher than 9, and our usage of `ts-node` is such that all we really care about is that we can call `register` and then `require` ts files, so it's less overhead if we start by not locking to a specific major.

tbh, we can probably lower the major to ~5; I don't think upgrading `ts-node` is a huge burden but probably want to require at most 8 rather than 9 as that major was released recently and drops <10 node support

---

All the options looks fine with me, but someone from the Jest team should decide on this (@SimenB)

---

Jest also doesn't support node<10, so that doesn't matter.

I do think `>=` makes sense, tho